<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beállítások</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	background-color: rgb(70, 70, 100);
	border: 2px solid #8f8f91;
	border-radius: 10px;
	color: white;
	font-size: 16px;
	font-weight: bold;
}</string>
    </property>
    <widget class="QPushButton" name="abortButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>490</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Mégse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveAndCloseButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>490</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Mentés</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>461</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: #3d3d7b;
	border: 2px solid #8f8f91;
	border-radius: 10px;
	color: white;
	font-size: 16px;
	font-weight: bold;
}</string>
     </property>
     <widget class="QFrame" name="profilePicture">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="changeProfilePictureButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Profilkép csere</string>
      </property>
     </widget>
     <widget class="QPushButton" name="restoreDefaultResultsButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
	background-color: rgb(255, 205, 201);
	color: grey;
}

*:hover {
	background-color: rgb(255, 102, 89);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Eredmények törlése</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteUserProfileButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
	background-color: rgb(255, 205, 201);
	color: grey;
}

*:hover {
	background-color: rgb(255, 102, 89);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Profil törlése</string>
      </property>
     </widget>
     <widget class="QComboBox" name="changeThemeBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteProfilePictureButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Profilkép törlése</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="changeUserAge">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="oldPassword">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Régi Jelszó</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="newPassword1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Új Jelszó</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="newPassword2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>360</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Új Jelszó Mégegyszer</string>
      </property>
     </widget>
     <widget class="QLabel" name="userAgeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>217</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	color: whitesmoke;
	border: none;
}</string>
      </property>
      <property name="text">
       <string>Életkor:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight</set>
      </property>
     </widget>
     <widget class="QLabel" name="oldPasswordLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>267</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	color: whitesmoke;
	border: none;
}</string>
      </property>
      <property name="text">
       <string>Régi Jelszó:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight</set>
      </property>
     </widget>
     <widget class="QLabel" name="newPassword1Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>317</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	color: whitesmoke;
	border: none;
}</string>
      </property>
      <property name="text">
       <string>Új Jelszó:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight</set>
      </property>
     </widget>
     <widget class="QLabel" name="newPassword2Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>367</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	color: whitesmoke;
	border: none;
}</string>
      </property>
      <property name="text">
       <string>Új Jelszó Mégegyszer:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight</set>
      </property>
     </widget>
     <widget class="QLabel" name="themeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>417</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	color: whitesmoke;
	border: none;
}</string>
      </property>
      <property name="text">
       <string>Téma:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
